application/Cache.php

```
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Cache utilities
+ */
+
+/**
+ * Purges all cached pages
+ *
+ * @param string $pageCacheDir page cache directory
+ */
+function purgeCachedPages($pageCacheDir)
+{
+    if (! is_dir($pageCacheDir)) {
+        return;
+    }
+
+    // TODO: check write access to the cache directory
+
+    $handler = opendir($pageCacheDir);
+    if ($handler == false) {
+        return;
+    }
+
+    while (($filename = readdir($handler)) !== false) {
+        if (endsWith($filename, '.cache')) {
+                unlink($pageCacheDir.'/'.$filename);
```

Will this fail silently? `$pageCacheDir` could be writable but not some `.cache` file inside (wrong permissions)